Assets/Scripts/FengGameManagerMKII.cs

```diff
@@ -626,16 +626,10 @@ private void core2()
             if (this.timeElapse > 1f)
             {
                 this.timeElapse--;
-                var content = Gamemode.GetGamemodeStatusTop((int) timeTotalServer, time);
-                if (Gamemode.Settings.TeamMode != TeamMode.Disabled)
-                {
-                    content +=
-                        $"\n<color=#00ffff>Cyan: {cyanKills}</color><color=#ff00ff>       Magenta: {magentaKills}</color>";
-                }
-
-                this.ShowHUDInfoTopCenter(content);
-                content = Gamemode.GetGamemodeStatusTopRight((int) timeTotalServer, time);
-                this.ShowHUDInfoTopRight(content);
+                //All of this has to go as it's a double from also the previous if
```

Comment:
So if you already discovered that this if statement does not have any impact, then please remove it.

Reply:
Ok i'll start reworking those than ;)